src/pages/physical-count-itemDetails/index.js

```diff
@@ -0,0 +1,633 @@
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      stockCountId: null,
+      siteId: null,
+      controllerId: null,
+      totalQty: null,
+      totalWeight: null,
+      status: 1,
+      SCStatus: null,
+      SCWIP: null,
+      EndofSiteStatus: null,
+      rows: [
+        {
+          id: 1,
+          sku: '',
+          itemId: null,
+          itemName: '',
+          countedQty: 0,
+          weight: 0,
+          metalPurity: 0
+        }
+      ]
+    },
+    maxAccess,
+    enableReinitialize: false,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      stockCountId: yup.string().required(),
+      siteId: yup.string().required(),
+      controllerId: yup.string().required(),
+      rows: yup.array().of(
+        yup.object().shape({
+          sku: yup.string().required(),
+          itemName: yup.string().required()
+        })
+      )
+    }),
+    onSubmit: async obj => {
+      const copy = { ...obj }
+
+      const items = obj?.rows?.map((item, index) => ({
+        ...item,
+        seqNo: index + 1,
+        siteId: copy.siteId,
+        stockCountId: copy.stockCountId,
+        controllerId: copy.controllerId
+      }))
+
+      const StockCountItemDetailPack = {
+        siteId: copy.siteId,
+        controllerId: copy.controllerId,
+        stockCountId: copy.stockCountId,
+        items: items
+      }
+
+      console.log('inSave')
+      console.log(StockCountItemDetailPack)
+
+      const response = await postRequest({
+        extension: SCRepository.StockCountItemDetail.set2,
+        record: JSON.stringify(StockCountItemDetailPack)
+      })
+
+      toast.success(platformLabels.Edited)
+      setEditMode(items.length > 0)
+      checkPhyStatus(copy.stockCountId, copy.siteId, copy.controllerId)
+
+      handleClick(items)
+    }
+  })
+
+  async function fetchGridData(stockCountId, siteId, controllerId) {
+    getDTDsku(stockCountId)
+
+    await getRequest({
+      extension: SCRepository.StockCountItemDetail.qry,
+      parameters: `_stockCountId=${stockCountId}&_siteId=${siteId}&_controllerId=${controllerId}`
+    })
+      .then(res => {
+        if (res.list) {
+          const modifiedList = res.list?.map((item, index) => ({
+            ...item,
+            id: index + 1,
+            metalPurity: item?.metalPurity ? item?.metalPurity : 0,
+            weight: item?.weight ? item?.weight : 0,
+            countedQty: item?.countedQty ? item?.countedQty : 0
+          }))
+          modifiedList.length > 0 && formik.setFieldValue('rows', modifiedList)
+        }
+
+        setEditMode(res.list.length > 0)
+        handleClick(res.list)
+      })
+      .catch(error => {})
+  }
+
+  useEffect(() => {
+    getSysChecks()
+  }, [])
+
+  useEffect(() => {
+    if (!formik.values.stockCountId) {
+      setSiteStore([])
+      setControllerStore([])
+      setFilteredItems([])
+      setEditMode(false)
+    }
+  }, [formik.values.stockCountId])
+
+  useEffect(() => {
```

Comment:
this useEffect and getTotals function can be removed, check how we sum the total for such field in grid in 'credit order' or any activity has footer summery